packages/react/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classnames from 'classnames';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Layout types
+ *
+ * @type {{'1-3': string[]}}
+ * @private
+ */
+const _types = {
+  '1-3': [`${prefix}--col-lg-4`, `${prefix}--col-lg-12`], // 1/4 - 3/4
+};
+
+/**
+ * Returns the child classes with the proper column class names
+ *
+ * @param {string} type layout type
+ * @param {object} children child nodes
+ * @returns {*} modified child content
+ * @private
+ */
+function _updateChild(type, children) {
+  const final = [];
+
+  children.map((child, i) => {
+    if (child.props['data-sticky'] === 'true') {
+      final.push(
+        <div
+          className={
+            _types[type] && _types[type][i] ? _types[type][i] : `${prefix}--col`
```

Comment:
Hmm, I think the default `bx--col` might be ok for now, we don't know how many other layouts will be introduced in the future.

Reply:
I'm thinking instead of maybe `bx--col` if we could just remove any extra children from the DOM if the count goes beyond what's defined in the `_types` array above. Just a thought.